packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -109,23 +88,34 @@ async function initProject(appName, options) {
 
     successMessage(fullAppFolder);
   } catch (err) {
-    console.log(`${chalk.red('Error:')} Failed to create ${appName}
-
-  ${err}
-`);
+    error(`Failed to create ${appName}`, String(err));
     process.exit(1);
   }
 }
 
 (async function main() {
   program
     .command('init', { isDefault: true })
-    .description('Bootstrap your app inside a new folder ‘app-name’')
-    .argument('[app-name]', 'App name')
+    .description([
+      'Bootstrap your app inside a new folder ‘my-app’',
+      '',
+      code('  create-contentful-app init my-app'),
+      '',
+      'or you can specify your own template',
+      '',
+      code('  create-contentful-app init my-app --templateSource "github:user/repo"'),
+      '',
+      `Contentful official templates are hosted at ${highlight('https://github.com/contentful/apps/tree/master/templates')}.`
+    ].join('\n'))
+    .argument('[app-name]', 'Name of the app')
     .option('--npm', 'Use NPM')
     .option('--yarn', 'Use Yarn')
-    .option('--javascript, -js')
-    .option('--typescript, -ts')
+    .option('--javascript, -js', 'Use default Javascript template')
+    .option('--typescript, -ts', 'Use default Typescript template')
+    .option('-t, --templateSource <url>', [
```

Comment:
Not really sure I like these options. I would've assumed that we have a `--templates, -t` for directly referencing a template from the `apps` repo. And something short like `--src` for a full url.